<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>우편물 요금제 확인</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>title_big.png</normaloff>title_big.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_renewdb">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>DB 갱신</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>요금제를 조회할 주소</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextEdit" name="edit_postfeeaddress">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="markdown">
         <string notr="true">http://www.koreapost.go.kr/kpost/subIndex/131.do

</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'맑은 고딕';&quot;&gt;http://www.koreapost.go.kr/kpost/subIndex/131.do&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>477</width>
         <height>670</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>통상우편물(규격)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTableWidget" name="table_nonstandardpost">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>무게</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>요금</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTableWidget" name="table_nextdaypost">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>수수료액</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>일반소포</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QTableWidget" name="table_registeredpost">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>수수료액</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>내용증명</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="table_standardpost">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>무게</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>요금</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>등기취급수수료(우편요금에 가산됨)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>익일특급수수료(우편요금에 가산됨)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>통상우편물(규격 외)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QTableWidget" name="table_proofofcontent">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>단 위</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>수수료액</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QTableWidget" name="table_normalpackage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>210</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>형태</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>요금</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QTableWidget" name="table_registeredpackage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>210</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>형태</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>요금</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>등기소포</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>소포요금 감액</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QTableWidget" name="table_packagediscount">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <column>
             <property name="text">
              <string>감액률</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>조건</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="action_closewindow"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>도움말</string>
    </property>
    <addaction name="action_visitpostfeeinfo"/>
    <addaction name="action_posttaginfo"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_closewindow">
   <property name="text">
    <string>창 닫기</string>
   </property>
  </action>
  <action name="action_openpostfeehelp">
   <property name="text">
    <string>국내 우편요금제도 도움말</string>
   </property>
  </action>
  <action name="action_visitpostfeepage">
   <property name="text">
    <string>우편요금제도 사이트 방문</string>
   </property>
  </action>
  <action name="action_posttaghelp">
   <property name="text">
    <string>우편물 태그 요령 안내</string>
   </property>
  </action>
  <action name="action_visitpostfeeinfo">
   <property name="text">
    <string>우편요금제도 사이트 방문</string>
   </property>
  </action>
  <action name="action_posttaginfo">
   <property name="text">
    <string>우편물 태그 요령 안내</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
